### ISSUE TITLE
domfuzz in Firefox 61

### ISSUE THREAD

**Issue by NicoPerezGarcia** [MozillaSecurity/domfuzz#15]
Hi,
Is it possible to run domfuzz in latest Firefox compilations? I have compiled Firefox 61 (mozilla-central) with `ac_add_options --enable-tests` and ASAN and when I run domfuzz it launches Firefox as usual but Firefox does not change the url. 
Do I need any extension?
Also I am wondering how domfuzz would work with the old functions of domFuzzlite add-on (like fuzzPriv.forceGC()) or if there is an extension available for that.

Thanks.

**Comment by jschwartzentruber** [comment]
The extension in this repo is unmaintained. It was broken out into https://github.com/MozillaSecurity/fuzzpriv/tree/legacy.

I have removed the copy here and made it a submodule instead. With this change, domfuzz should still be working with mozilla-central.

**Comment by NicoPerezGarcia** [comment]
Thanks for you replay,

I am still having issues trying to make it working. I am getting this error:
`1523555900457	addons.xpi-utils	WARN	addMetadata: Add-on domfuzz@squarefree.com is invalid: Error: File /<user_folder>/trees/domfuzz/dom/automation/../extension does not contain a valid install manifest (resource://gre/modules/addons/XPIInstall.jsm:889:11) JS Stack trace: loadManifest@XPIInstall.jsm:889:11`

Checking the "extension" folder, it is empty. This could be the problem that I have related to changing the url (after launching, Firefox doesn't do anything.). Am I right?

And about the domFuzzlite add-on extension, how should I install it? Is there any folder in domFuzz to move it? Because after launching Firefox, I have tried to call fuzzPriv.GC() but it raises "fuzzPriv is undefined".
